packages/gapic-node-templating/templates/bootstrap-templates/samples/quickstart.js

```diff
@@ -16,9 +16,9 @@
 'use strict';
 
 function main() {
-  // [START quickstart]
+  // [START {{name}}_quickstart]
   console.log('Missing samples and tests');
-  // [END quickstart]
+  // [END {{name}}_quickstart]
 }
```

Comment:
To let snippet-bot ignore this file, add `.github/snippet-bot.yml` with the following content:

```yaml
ignoreFiles:
  - "packages/gapic-node-templating/templates/bootstrap-templates/samples/quickstart.js"
```